src/app/design/frontend/base/default/template/shopgate/cloudapi/content/customer/account/login.phtml

```diff
@@ -0,0 +1,5 @@
+<div class="back-to-login">
+    <button type="button" class="button"
+            title="<?php echo Mage::helper('shopgate_cloudapi')->quoteEscape($this->__('Back to login')) ?>" name="send" id="send2">
+        <span><span><?php echo $this->__('Back to login') ?></span></span></button>
+</div>
```

Comment:
this seems to be not going the right direction with the template locations, can you change the locations of these templates while we are in these early stages? e.g.
```
frontend/base/default/template/shopgate/cloudapi/customer/account/login/header.phtml
frontend/base/default/template/shopgate/cloudapi/customer/account/login/content.phtml
```
Adjust `modman` if necessary.

Reply:
you are right, looks cleaner now. pls. check.